settings.sample.py

```diff
@@ -1,17 +1,19 @@
+import os
+
 # Webserver configuration #
 ###########################
 
 # gunicorn config
-bind = '127.0.0.1:5000'
-errorlog = 'lintreview.error.log'
-accesslog = 'lintreview.access.log'
-debug = True
-loglevel = 'debug'
+bind = os.environ.get('LINTREVIEW_GUNICORN_BIND', '127.0.0.1:5000')
+errorlog = os.environ.get('GUNICORN_LOG_ERROR', 'lintreview.error.log')
```

Comment:
Honestly, it was because it made the line too long :-)
If the consensus is to prefix all of the environment with LINTREVIEW_ I will change it and wrap the line.